challenge.js

```diff
@@ -57,5 +57,5 @@
  */
 
 // ANSWER
-
+// To attach an element as a child of an element, you would refer to the element you want to add it to and use the method .appendChild. (element.appendChild(<elementYouWantToAttach>))(?)
```

Comment:
That's right. This is great for elements we've created in our JS using `document.createElement` (and off-DOM element) or an on-DOM element we detached from the DOM and want to reattach elsewhere. Here's a snippet that demonstrates this usage:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1